examples/nginx-with-rspamd/docker-compose.yml

```diff
@@ -0,0 +1,82 @@
+version: "3.5"
+
+services:
+  db:
+    image: mariadb:10.5
+    container_name: anonaddy_db
+    command:
+      - "mysqld"
+      - "--character-set-server=utf8mb4"
+      - "--collation-server=utf8mb4_unicode_ci"
+    volumes:
+      - "./db:/var/lib/mysql"
+    environment:
+      - "MYSQL_ALLOW_EMPTY_PASSWORD=yes"
+      - "MYSQL_DATABASE"
+      - "MYSQL_USER"
+      - "MYSQL_PASSWORD"
+    restart: always
+
+  redis:
+    image: redis:4.0-alpine
+    container_name: anonaddy_redis
+    restart: always
+
+  anonaddy:
+    image: anonaddy/anonaddy:latest
+    container_name: anonaddy
+    depends_on:
+      - db
+      - redis
+    ports:
+      - target: 25
+        published: 25
+        protocol: tcp
+      - target: 8000
+        published: 8000
+        protocol: tcp
+    volumes:
+      - "./data:/data"
+    env_file:
+      - "./anonaddy.env"
+    environment:
+      - "DB_HOST=db"
+      - "DB_DATABASE=${MYSQL_DATABASE}"
+      - "DB_USERNAME=${MYSQL_USER}"
+      - "DB_PASSWORD=${MYSQL_PASSWORD}"
+      - "REDIS_HOST=redis"
+    restart: always
+
+  nginx:
+    image: nginx:1.20.1-alpine
+    container_name: anonaddy_nginx
+    restart: unless-stopped
+    ports:
+      - '443:443'
+    volumes:
+      - ./nginx/dhparam.pem:/etc/ssl/dhparam.pem
+      - ./nginx/templates:/etc/nginx/templates
+      - /etc/letsencrypt:/etc/letsencrypt
+    environment:
+      - NGINX_HOST=example.com
+      - NGINX_PROXY=anonaddy:8000
+    depends_on:
+      - anonaddy
+      - rspamd
```

Comment:
Is nginx necessary for Rspamd to work? If not I think it should be done in a follow-up instead to avoid confusion so the example could be called `rspamd` and the example for nginx `nginx`.

Reply:
not necessary. this is just a simplified setup and one i can verify works. nginx solves coordinating ssl and proxy to both web services (anonaddy / rspamd web ui). we can simplify it first and then perhaps add an nginx example as an additional example later. good idea.